docs/release/osg-23.md

```diff
@@ -29,6 +29,19 @@ This issue is still under investigation.
 Latest News
 -----------
 
+!!! warning "Incoming changes to `osg-ca-certs`"
+    To address an issue with certificates issued by SHA1 CAs on EL9 systems and EL8 systems configured to use the
+    `FUTURE` system crypto policy,
+    the OSG Software Team intends to release a fix to the `osg-ca-certs` package during the **week of November 27,
+    2023** (<https://opensciencegrid.atlassian.net/browse/SOFTWARE-5745>).
+
+    -   If you are running OSG-supported software (e.g., HTCondor, XRootD),
+        update to the new version of `osg-ca-certs`.
+
+    -   If you are running unsupported, java-based software (e.g., dCache) with `osg-ca-certs`,
+        install `osg-ca-certs-java` instead of `osg-ca-certs`.
+        `osg-ca-certs-java` is compatible with java-based software but does not include the aforementioned fix.
```

Comment:
```suggestion
    -   If you are running non-OSG-supported, Java-based software (e.g., dCache) with `osg-ca-certs`,
        install `osg-ca-certs-java` instead of `osg-ca-certs`.
        `osg-ca-certs-java` is compatible with Java-based software but does not include the aforementioned fix.
```